packages/server/src/course/course.controller.ts

```diff
@@ -133,18 +132,23 @@ export class CourseController {
       );
     }
 
-    if (userCourseModel.role === Role.PROFESSOR) {
-      course.queues = await async.filter(
-        course.queues,
-        async (q) => (await q.checkIsOpen()) || q.isProfessorQueue,
+    if(userCourseModel.role == Role.PROFESSOR){
+      course.queues = await async.filter(course.queues, async (q) =>
+          !q.isDisabled,
       );
-    } else {
-      course.queues = await async.filter(
-        course.queues,
-        async (q) => await q.checkIsOpen(),
+    }
+    else if(userCourseModel.role == Role.TA){
+      course.queues = await async.filter(course.queues, async (q) =>
+          !q.isDisabled && !q.isProfessorQueue,
```

Comment:
Thinking back on this I feel like it probably makes sense to only show queues that are currently open (have staff members), since it seems like that's what we were doing with the UX before. Still open to hearing other opinions tho

Reply:
done